integration-tests/devtools/src/test/java/io/quarkus/devtools/codestarts/QuarkusCodestartRunIT.java

```diff
@@ -93,6 +94,49 @@ public void generateAzureFunctionsHttpExampleProjectRun() throws Exception {
                 Collections.emptyMap());
     }
 
+    @Test
+    public void generateGradleWrapperGithubAction() throws Exception {
```

Comment:
I think those tests would fit better in `QuarkusCodestartGenerationTest`, `QuarkusCodestartRunIT` is there to run the generated codestarts..